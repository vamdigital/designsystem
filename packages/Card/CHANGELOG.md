# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.0.3](https://github.com-vamdigital/vamdigital/designsystem/compare/@vam/card@0.0.2...@vam/card@0.0.3) (2019-08-31)

**Note:** Version bump only for package @vam/card





## [0.0.2](https://github.com-vamdigital/vamdigital/designsystem/compare/@vam/card@0.0.1...@vam/card@0.0.2) (2019-08-31)

**Note:** Version bump only for package @vam/card





## 0.0.1 (2019-08-31)

**Note:** Version bump only for package @vam/card
