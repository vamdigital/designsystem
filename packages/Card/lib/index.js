

Object.defineProperty(exports, '__esModule', {
  value: true
});
exports.default = void 0;

var _react = _interopRequireDefault(require('react'));

function _interopRequireDefault(obj) {
  return obj && obj.__esModule ? obj : { default: obj };
}

const Card = () => _react.default.createElement('div', null, 'Card');

var _default = Card;
exports.default = _default;
//# sourceMappingURL=index.js.map
