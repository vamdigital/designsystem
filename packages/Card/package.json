{
  "name": "@vam/card",
  "version": "0.0.0",
  "description": "Card Component",
  "main": "/lib/index.js",
  "author": "VAM",
  "license": "MIT",
  "peerDependencies": {
    "react": "^16.9.0"
  }
}
