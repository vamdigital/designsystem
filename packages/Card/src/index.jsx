import React from 'react';

const Card = () => {
  return <div>I am card component</div>;
};

export default Card;
