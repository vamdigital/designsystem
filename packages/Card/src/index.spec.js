import React from 'react';
import { shallow } from 'enzyme';
import Card from './index';

describe('<Card />', () => {
  let wrapper;
  beforeAll(() => {
    wrapper = shallow(<Card />);
  });
  it('should render Card Component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
