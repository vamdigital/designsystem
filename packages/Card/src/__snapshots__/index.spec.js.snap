// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should render Card Component 1`] = `
<div>
  I am card component
</div>
`;
